### ISSUE TITLE
[2019/08/07] Machine Learning 輪講

### ISSUE THREAD

**Issue by agatan** [wantedly/machine-learning-round-table#13]
## Why

Machine Learning 輪講は最新の技術や論文を追うことで、エンジニアが「技術で解決できること」のレベルをあげていくことを目的にした会です。

prev. #12 

## What

話したいことがある人はここにコメントしましょう！
面白いものを見つけた時点でとりあえず話すという宣言だけでもしましょう！

**Comment by agatan** [comment]
## Variational Pretraining for Semi-supervised Text Classification

- https://arxiv.org/abs/1906.02242v1
- ACL 2019

提案手法の名前がかっこいい（VAMPIRE）。
少量の unlabeled データしかないという状況で計算効率・精度の良い事前学習をするための手法を提案（事前学習 → downstream classifier に context embedding をわたす）。

BERT とか確かにめっちゃ良いけど計算資源やデータを大量に必要とするし fine-tune すらそれなりに大変。データ量少なくてもうまく動いてほしいし、downstream classifier の精度は in-domain なドキュメントで事前学習したほうが性能がよい。
VAMPIRE は各 document を bag-of-words で表し、VAE を利用して事前学習を行う。

**Comment by agatan** [comment]
## あんまりちゃんと読んでないけど面白そうなものシリーズ

- Bayesian Uncertainty Estimation for Batch Normalized Deep Networks
    - https://arxiv.org/abs/1802.06455
- Differentiable Sorting using Optimal Transport:The Sinkhorn CDF and Quantile Operator
    - https://arxiv.org/abs/1905.11885v1
- https://github.com/gnes-ai/gnes
    - なんでもベクトルにしてくれたらいい感じに検索する君
- https://github.com/williamFalcon/pytorch-lightning
    - pytorch 版 keras 的な役割。めっちゃ便利そう。
    - pytorch 本体の ecosystem に加わるという噂を観測した。
- https://twitter.com/fchollet/status/1158858005917515776
    - > TensorFlow 2.0.0 is coming very soon. Be excited!
    - もう rc でた？

**Comment by tkgsn** [comment]
センチメント分析まとめ

-語彙ベース
極性辞書が必要
-機械学習ベース
ラベル付けが必要
株価変動

極性辞書構築
辞書ベース
→語彙ネットワークを辞書から生成（辞書が必要）
コーパスベース
-PMI行列を使って類似度計算→伝搬
-単語分散表現を使って類似度計算→伝搬
-ネットワークの表現学習(係り受けを意識したもの(LINE))→伝搬

https://www.ai-gakkai.or.jp/jsai2017/webprogram/2017/pdf/894.pdf
LDAで使う
https://sigfin.org/?plugin=attach&refer=SIG-FIN-017-07&openfile=SIG-FIN-017-07.pdf

**Comment by awakia** [comment]
## 準同型暗号でDeepLearning

[準同型暗号 - Wikipedia](https://ja.wikipedia.org/wiki/%E6%BA%96%E5%90%8C%E5%9E%8B%E6%9A%97%E5%8F%B7)
[Homomorphic encryption - Wikipedia](https://en.wikipedia.org/wiki/Homomorphic_encryption)
[https://arxiv.org/pdf/1811.00778.pdf](https://arxiv.org/pdf/1811.00778.pdf) 

掛け算と足し算しかできないのでActivationは単純な２乗でやって
MNISTは99%, CIFAR10は77%ぐらいまで達成している
